### PR TITLE
 SvelteKit release candidate migration: documentation

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Some documentation in this repo has become outdated with the recent SvelteKit release candidate migration.

- [x] README.md
- [x] src/lib/public/legacy/README.md
- [x] CONTRIBUTING.md
- [x] ARCHITECTURE.md
- [x] GETTING_STARTED.md

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -5,55 +5,43 @@ This project was created using a library called [SvelteKit][sveltekit_home] whic
 ## `/src/routes/` 📁
 
 The website's pages are located under the [`routes` directory](src/routes).
-In that directory, each file represents one page of the website.
+In that directory, each file with a prefix <kbd>+</kbd> represents endpoint of the website.
```
Comment (EthanThatOneKid) [inline ARCHITECTURE.md]:
```suggestion
In that directory, each file with a prefix <kbd>+</kbd> represents an endpoint of the website.
```

File: ARCHITECTURE.md [hunk 2]
```diff
@@ -5,55 +5,43 @@ This project was created using a library called [SvelteKit][sveltekit_home] whic
...
+| `/src/routes/about/+page.svelte` | `https://acmcsuf.com/about/` |
+| `/src/routes/paths/+page.svelte` | `https://acmcsuf.com/paths/` |
 
 > ℹ more info: <https://kit.svelte.dev/docs/routing#advanced-routing>
 
-### `/src/routes/__layout.svelte` 📄
+### `/src/routes/+layout.svelte` 📄
 
-The [`__layout.svelte file`](src/routes/__layout.svelte) at the root of the `/routes` directory is used in the background of every page of the site.
+The [`layout.svelte file`](src/routes/+layout.svelte) at the root of the `/routes` directory is used in the background of every page of the site.
```
Comment (EthanThatOneKid) [inline ARCHITECTURE.md]:
```suggestion
The [`+layout.svelte file`](src/routes/+layout.svelte) at the root of the `/routes` directory is used in the background of every page of the site.
```

File: ARCHITECTURE.md [hunk 3]
```diff
@@ -5,55 +5,43 @@ This project was created using a library called [SvelteKit][sveltekit_home] whic
...
-
-There are a couple of directories in the components directory that do not fit the `/lib/components/[route_name]/[component_name].svelte` pattern.
-One of them is the [`/lib/components/utils/` directory](src/lib/components/utils).
-This directory contains utility components that are used by multiple routes.
-This includes `acm-button.svelte` and `acm-select.svelte`, for example.
-
-### `/src/lib/components/icons/` 📁
-
-The [`/lib/components/icons/` directory](src/lib/components/icons) contains several Svelte files that contain SVG data.
+The [`/lib/components/sbg/` directory](src/lib/components/svg) contains several Svelte files that contain SVG data.
```
Comment (EthanThatOneKid) [inline ARCHITECTURE.md]:
```suggestion
The [`/lib/components/svg/` directory](src/lib/components/svg) contains several Svelte files that contain SVG data.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -116,13 +120,13 @@ Other environment variables are needed to provide information that allows the pr
 
 ```sh
 # General discussion on GitHub: <https://github.com/EthanThatOneKid/acmcsuf.com/discussions/categories/newsletter>
-VITE_GH_DISCUSSION_CATEGORY_ID=DIC_kwDOE7ysSc4CAC0o
+GH_DISCUSSION_CATEGORY_ID=DIC_kwDOE7ysSc4CAC0o
 
 # Algo discussion on GitHub: <https://github.com/EthanThatOneKid/acmcsuf.com/discussions/categories/algo>
-VITE_GH_ALGO_CATEGORY_ID=DIC_kwDOE7ysSc4COe-m
+GH_ALGO_CATEGORY_ID=DIC_kwDOE7ysSc4COe-m
```
Comment (EthanThatOneKid) [inline CONTRIBUTING.md]: There is no need to mention `GH_ALGO_CATEGORY_ID`, an unused environment variable.

**Review by EthanThatOneKid — COMMENTED** [review]
I noticed a few typos.

File: ARCHITECTURE.md (hunk 2 above)
Comment (EthanThatOneKid) [reply ARCHITECTURE.md]: Done.

File: ARCHITECTURE.md (hunk 3 above)
Comment (EthanThatOneKid) [reply ARCHITECTURE.md]: Done.

File: CONTRIBUTING.md (hunk 4 above)
Comment (EthanThatOneKid) [reply CONTRIBUTING.md]: Done.

File: ARCHITECTURE.md (hunk 1 above)
Comment (EthanThatOneKid) [reply ARCHITECTURE.md]: Done.

**Review by jaasonw — APPROVED** [review]
👍

**Comment by EthanThatOneKid** [comment]
Replaced by mega-PR #577.

**Pull request closed without merging**